inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -419,7 +424,11 @@ StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath, const std
         }
     }
 
+#ifdef ENABLE_V7_SERIALIZE
+    return network->serialize(xmlPath, binPath, resp);
+#else
     return DescriptionBuffer(NOT_IMPLEMENTED, resp) << "The serialize for IR v10 is not implemented";
+#endif
```

Comment:
> "The serialize for IR v10 is not implemented"

Why we disable this feature while there is no alternatives?

Reply:
Because IR v7 and older are deprecated. What customer can do with IR v7 which would be generated here?

Reply:
So why we don't upgrade `serialize` method to produce IRv10?

Reply:
On the other hand I don't mind to disable it by default. Please just note that this feature still might be useful and it'd be nice to have actual serialize as well.

Reply:
> So why we don't upgrade serialize method to produce IRv10?

Are you ready to contribute? :)
This is in plans for the next release

Reply:
v10 serializer should use nGraph mechanism which is not still fully integrated to the Inference Engine.

Reply:
@ilya-lavrenov not yet :) so that's why "I don't mind to disable it by default" but keep at source level.